lib/schemas/ResourceSchema.js

```diff
@@ -151,26 +189,12 @@ module.exports = makeSchema(
         maxLength: 255
       },
       // TODO: do we need to break these all apart too? :-/
-      get: {
-        description: ResourceMethodGetSchema.schema.description,
-        $ref: ResourceMethodGetSchema.id
-      },
-      hook: {
-        description: ResourceMethodHookSchema.schema.description,
-        $ref: ResourceMethodHookSchema.id
-      },
-      list: {
-        description: ResourceMethodListSchema.schema.description,
-        $ref: ResourceMethodListSchema.id
-      },
-      search: {
-        description: ResourceMethodSearchSchema.schema.description,
-        $ref: ResourceMethodSearchSchema.id
-      },
-      create: {
-        description: ResourceMethodCreateSchema.schema.description,
-        $ref: ResourceMethodCreateSchema.id
-      },
+      get: makeSingleOrNestedMultiple(ResourceMethodGetSchema),
+      hook: makeSingleOrNestedMultiple(ResourceMethodHookSchema),
+      list: makeSingleOrNestedMultiple(ResourceMethodListSchema),
+      search: makeSingleOrNestedMultiple(ResourceMethodSearchSchema),
+      create: makeSingleOrNestedMultiple(ResourceMethodCreateSchema),
+      misc: makeSingleOrNestedMultiple(ResourceMethodMiscSchema),
```

Comment:
The new `misc` pseudo type would just go into actions, same place where create goes and where misc stuff goes today.

Reply:
Is there a more self-documenting name we could give this, like `additionalActions` or `additionalOperations`?

Reply:
Hmmm, agree it isn't perfect but nothing is jumping out at me. Since it is mostly local I am not too worried about it.